specs/fault-proof/stage-one/bond-incentives.md

```diff
@@ -148,10 +148,16 @@ checks that the current `block.timestamp` is greater than the timestamp on the w
 seconds and reverts if not. It also confirms that the amount being withdrawn is less than the amount in the withdrawal
 request. Before completing the withdrawal, it reduces the amount contained within the withdrawal request. The original
 `withdraw(wad)` function becomes an alias for `withdraw(msg.sender, wad)`.
-`withdraw(guy,wad)` will not be callable when `SuperchainConfig.paused()` is `true`.
-- `DelayedWETH` has a `hold()` function that allows the `owner()` address to give itself an allowance from any address.
+`withdraw(guy,wad)` will not be callable when `SuperchainConfig.paused()` is `true`. Also, `withdraw(guy,wad)` is not callable
+when `delayedWethPaused` is `true`, unless the caller is the `owner()`.
+- `DelayedWETH` has a `setDelayedWethPaused(bool)` function that allows the `owner()` address to either pause or unpause 
+withdrawals and transfers.
+- `DelayedWETH` has a `hold()` function that allows the `owner()` address to, for any holder, give itself an allowance and
```

Comment:
We should include the arguments for `hold` here for consistency. I also wonder if there's a better name than `hold` given it takes the funds, not just holding them.  Maybe `reallocate`? Not sure how much it's worth worrying about this.

Reply:
Probably wouldn't bother worrying about it

Reply:
Agree on the arguments

Reply:
Fixed in ce0e3e1